cli/lib/kontena/cli/stacks/validate_command.rb

```diff
@@ -19,6 +19,7 @@ class ValidateCommand < Kontena::Command
     option '--dependency-tree', :flag, "Show dependency tree"
     option '--[no-]dependencies', :flag, "Validate dependencies", default: true
     option '--parent-name', '[PARENT_NAME]', "Set parent name", hidden: true
+    option '--output-json', :flag, "Output API JSON"
```

Comment:
Maybe just `--json` or for consistency: `--format json` as in `kontena master config dump`:

```
    -f, --format [FORMAT]         Specify output format (json, yaml) (default: guess from PATH or json)
```

Reply:
It's not just the syntax, but the actual structure and semantics of the output changes (stack YAML vs API JSON).

Maybe this should be something like `--api-json` then?

Reply:
How about `--format api-json` ?

Reply:
Changed from `--output-json` to `--format=yaml|api-json`